go/chat/utils/kbfs_path_utils.go

```diff
@@ -55,11 +57,11 @@ func matchKBFSPathOuter(body string) (outerMatches []outerMatch) {
 		case indices[4] > 0:
 			outerMatches = append(outerMatches, outerMatch{
 				matchStartIndex: indices[2],
-				wholeMatch:      body[indices[2]:indices[3]],
+				wholeMatch:      strings.TrimRight(body[indices[2]:indices[3]], unquotedTrailingTrimCutset),
```

Comment:
You might be able to do this without defining your own punctuation string with something like:

```go
strings.TrimRightFunc(..., unicode.IsPunct)
```

but i can't figure out exactly what `IsPunct` matches.

Reply:
Nice! Didn't know about that. It'd cover `/` and `\` which I don't want to remove in `wholeMatch`, but I guess I could have a function that skips those before calling `unicode.IsPunct`.